### PR TITLE
Code examples for episode 016 & 017

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: ep016/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# [pretty python profiling (intermediate)](https://youtu.be/ey_P64E34g0)
+
+Today I show how I figure out what's slow in a python application and some cool visualization tools!
```
Comment (int3l) [inline ep016/README.md:3]: Do you want the extra info from the video description to be here as well?

Comment (asottile) [reply ep016/README.md:3]: up to you

**Review by asottile — APPROVED** [review]

File: ep016/README.md (hunk 1 above)
Comment (int3l) [reply ep016/README.md:3]: Links are subject that can change in the future and they are at two places already - in the video & in the video description, so I guess this is enough redundancy :D

**Pull request merged**